test/functionalTest/cases/2998_null_not_working_in_q_and_mq_filter/null_not_working_in_mq_for_subscription.test

```diff
@@ -0,0 +1,258 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
We are aproaching to 1000 .test (wow! :O). Please, modify testHarness.sh in this PR to add a new digit to the counter.

Reply:
Fixed in ba413e8